docs/content/monitoring.md

```diff
@@ -51,7 +51,7 @@ The Prometheus endpoint exports Go runtime metrics as well as HTTP request laten
 | go_memstats_alloc_bytes_total | counter | Total number of bytes allocated, even if freed. | STABLE |
 | go_memstats_buck_hash_sys_bytes | gauge | Number of bytes used by the profiling bucket hash table. | STABLE |
 | go_memstats_frees_total | counter | Total number of frees. | STABLE |
-| go_memstats_gc_cpu_fraction | gauge | The fraction of this program's available CPU time used by the GC since the program started. | STABLE |
+| ~~go_memstats_gc_cpu_fraction~~ | ~~gauge~~ | ~~The fraction of this program's available CPU time used by the GC since the program started.~~ | REMOVED ([Prometheus issue link](https://github.com/prometheus/client_golang/issues/1500)) |
```

Comment:
We can probably just remove the row and include the prom issue details in the commit message.

Reply:
Sounds good to me! :slightly_smiling_face:

Reply:
Updated commit message, and removed that line in the table.